src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -58,6 +58,7 @@ public function exists($name)
      * Otherwise the template is located using the locator from the twig library.
      *
      * @param string|TemplateReferenceInterface $template The template
+     * @param bool $throw Throw \Twig_Error_Loader exception if template is not found
```

Comment:
And I would change the comment to something like "When true, a `\Twig_Error_Loader` exception will be thrown if a template could not be found".